src/app/core/entity/entity-actions/entity-actions.service.ts

```diff
@@ -26,7 +26,7 @@ export class EntityActionsService {
     private entityAnonymize: EntityAnonymizeService,
   ) {}
 
-  private showSnackbarConfirmation(
+  showSnackbarConfirmationWithUndo(
```

Comment:
or shall we even load the original (for undo) from db here and include entitymapper.save as well , so that this can become a complete replacement for EntityMapper.save ?